webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -140,25 +251,75 @@ const DeferToProduction = (): JSX.Element => {
                 />
               </Label>
             </FormGroup>
-            <FormGroup>
-              <Stack>
+            <Label>Save your file location</Label>
+            <FormGroup check className={classes.pathSelection}>
+              <div className={classes.pathSelectionRow}>
                 <Label
+                  check
                   for="manifestPath"
```

Comment:
this `for` attribute should match the id of the input radio element to trigger click event on label. Otherwise remove the for attribute and just leave the labelwrapping the input. This will give better user experience. currently when i click the label it does not select the radio button